README.md

```diff
@@ -38,82 +38,37 @@ npm run test
+### When the type is missing
 
-1. First check we have the spec name `Intersection Observer` in `inputfiles/idlSources.json`. If not, add it.
-2. Run `npm run fetch-idl "Intersection Observer" && npm run build && npm run baseline-accept`.
+It's possible that the automated algorithm decided that it's not well supported by browsers and thus removed it. Say we we want to add a new interface named `Foo`. Check if there is a document about that interface in [MDN](https://developer.mozilla.org/). If there is, see the browser compatibility section and check whether it's supported by two or more browser engines. (Note that Chromium-based browsers use the same browser engine and thus support from them counts as a single support.)
 
-If the above didn't fix the type issues, we can fix them via json files as a last resort.
-There are three json files that are typically used to alter the type generation: `addedTypes.json`, `overridingTypes.json`, and `removedTypes.json`.
-`comments.json` can used to add comments to the types.
-Finally, `knownTypes.json` determine which types are available in a certain environment in case it couldn't be automatically determined.
+If all the conditions are fulfilled, it could be that the type is incorrectly removed by `inputfiles/removedTypes.jsonc`. Try finding and removing the relevant item there and run `npm run build && npm run baseline-accept`.
 
-The format of each file can be inferred from their existing content.
+If conditions are not fulfilled but you think MDN is wrong, please file an issue at https://github.com/mdn/browser-compat-data/issues/. The type will be automatically added in a few weeks when MDN fixes their data.
 
-The common steps to send a pull request are:
+### When the type exists but is wrong
 
-0. Open or refer to an issue in the [TypeScript repo](https://github.com/Microsoft/TypeScript) which might get moved back to this repo.
-1. Update an IDL or add missing elements to `inputfiles/addedTypes.json`, overriding elements to `inputfiles/overridingTypes.json`, or elements to remove to `inputfiles/removedTypes.json`.
-2. Run the build script locally to obtain new `dom.generated.d.ts` and `webworker.generated.d.ts`.
-3. Update the files in the `baselines` folder using the newly generated files under `generated` folder (`npm run baseline-accept`).
+It's possible that the type is too specific or too general. First you need to check whether `inputfiles/overridingTypes.jsonc` or `inputfiles/addedTypes.jsonc` have a relevant item, which can be fixed if exists. If they don't, add one in `overridingTypes.jsonc`. Run `npm run build && npm run baseline-accept` to make sure the resulting changes are what you want.
 
-### What are the TypeScript team's heuristics for PRs to the DOM APIs
-
-Changes to this repo can have pretty drastic ecosystem effects, because these types are included by default in TypeScript.
```

Comment:
This paragraph is mostly still true, right? I know we are less conservative than before, but the idea is that every update to `@types/web` will *eventually* ship with Typescript by default means that there are some changes that will not be OK until there's a way to publish other packages from here.

Reply:
Yes, maybe it's good to keep the sentence for maintainers. It has minimal meaning for contributors though, since nearly all changes will now be from webref and MDN rather than manual PRs.

Reply:
>I know we are less conservative than before

I'm not sure, as quite a lot types are removed for lack of broad browser support?